distage/distage-testkit-core/src/main/scala/izumi/distage/testkit/services/dstest/SpecEnvironment.scala

```diff
@@ -20,6 +20,7 @@ trait SpecEnvironment {
 
   def contextOptions: PlanningOptions
 
+  def configOverrides: AppConfig => AppConfig
```

Comment:
IMHO this is also unnecessary, because of `override` + `ConfigLoader#map`

Reply:
Yup, but `ConfigLoader#map` requires SpecEnvironment override to apply. Also, `moduleOverrides` (if I understood you properly) can not update config sources and point to another folder/file.